test/functionality/abstract-interpretation/data-frame/data-frame.ts

```diff
@@ -32,56 +32,74 @@ export const DataFrameTestOverapproximation = {
 	rows:     DomainMatchingType.Overapproximation
 };
 
+interface CriterionTestEntry {
+	criterion:  SingleSlicingCriterion,
+	value:      DataFrameDomain,
+	node:       RSymbol<object>,
+	lineNumber: number
+}
+
 export function assertDataFrameDomain(
 	shell: RShell,
 	code: string,
-	criterion: SingleSlicingCriterion,
-	expected: DataFrameDomain,
+	expected: [SingleSlicingCriterion, DataFrameDomain][],
 	name: string = code
```

Comment:
You can udpate the name to allow for a `TestLabel` to support [`label`ing](https://github.com/flowr-analysis/flowr/wiki/Linting-and-Testing#writing-a-test) your tests. You can even add a new category/context for absint.